generators/init/templates/.eslintignore.ejs

```diff
@@ -0,0 +1,29 @@
+<%#
```

Comment:
Let's drop duplicated files.
`jhipsterTemplatesFolders` in the constructor should allow a transparent migration.

Reply:
At the moment, I'd prefer to keep this, although it's duplicated. It's for keeping this sub generator totally stand alone and not depend on other workflows
In any case, this file should not move a lot, so it should be OK